primitive-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "primitive-types"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
unfortunately, updating `scale-info` is a breaking change, so it has to 0.10.0, which will result in major bump in most of the crates in the workspace, sorry about that :/